3_javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,59 @@
+let name = "";
```

Comment:
one more thing, I would rename this variable, because ' name' is deprecated. Or you can just move the variable inside the func getReply